Dockerfile

```diff
@@ -0,0 +1,10 @@
+
+from python:3.6
+RUN pip install --upgrade pip && pip install \
+  prompt-toolkit==3.0.29 \
+   \
+  pygments
+RUN mkdir /xonsh
+WORKDIR /xonsh
+ADD ./ ./
+RUN python setup.py install
```

Comment:
I don't think this Dockerfile needs to be included in here

Reply:
Hi, maintainers,

Thankyou for your review. I was trying to run this command `pre-commit run --all-files` for the last few days. Because when I run this command, I just got `pre-commit: command not found`. And I'm still trying to install it because the installation always gets stuck. I'm trying to fix style errors by other methods.

Reply:
You should be able to `pip install pre-commit`